src/util/time.h

```diff
@@ -136,6 +136,12 @@ std::string FormatISO8601DateTime(int64_t nTime);
 std::string FormatISO8601Date(int64_t nTime);
 std::optional<int64_t> ParseISO8601DateTime(std::string_view str);
 
+/**
+ * RFC7231 formatting https://www.rfc-editor.org/rfc/rfc7231#section-7.1.1.1
+ * Used in HTTP/1.1 responses
+ */
+std::string FormatRFC7231DateTime(int64_t nTime);
```

Comment:
Not sure if we should have the RFC in the name if it's ambiguous where the format is specified (see above). Maybe `FormatHTTPDate` or something like that would be better.